Print which methods are being invalidated

This is a really unhelpful error message:

```
Run exit 1
  exit 1
  shell: /usr/bin/bash -e {0}
  env:
    JULIA_PKG_SERVER_REGISTRY_PREFERENCE: eager
Error: Process completed with exit code 1.
```

--- a/action.yml
+++ b/action.yml
@@ -33,7 +33,7 @@ runs:
       shell: bash
       
     - name: Install SnoopCompile tools
-      run: julia --project -e 'using Pkg; Pkg.add(["SnoopCompileCore", "SnoopCompile"])'
+      run: julia --project -e 'using Pkg; Pkg.add(["SnoopCompileCore", "SnoopCompile", "PrettyTables"])'
       shell: bash
     - name: Load package on branch
       id: invs
@@ -47,6 +47,14 @@ runs:
         inv_deps = inv_total - inv_owned
         
         @show inv_total, inv_deps
+
+        # Report invalidations summary:
+        using PrettyTables  # needed for `report_invalidations` to be defined
+        SnoopCompile.report_invalidations(;
+             invalidations,
+             process_filename = x -> last(split(x, ".julia/packages/")),
+             n_rows = 0,  # no-limit (show all invalidations)
+        )
 
         # Set outputs
         open(ENV["GITHUB_OUTPUT"], "a") do io

